KsApi/models/Comment.swift

```diff
@@ -32,13 +25,13 @@ public struct Comment {
     public var imageUrl: String
     public var isCreator: Bool
     public var name: String
-  }
 
-  public enum AuthorBadge: String {
-    case creator
-    case backer
-    case superbacker
-    case you // This doesn't exist on GraphComment, but would be mapped with current user id and author id
+    public enum AuthorBadge: String, Decodable {
```

Comment:
I think if we're going to nest `AuthorBadge` inside `Author` then it can just be `Badge` so that when we use it it's `Author.Badge` instead of `Author.AuthorBadge`.